docs/src/tutorial/external.rst

```diff
@@ -103,7 +138,19 @@ You can now make it clear which of the two arguments does what in
 your call, thus avoiding any ambiguities and often making your code
 more readable:
 
-.. literalinclude:: ../../examples/tutorial/external/keyword_args_call.pyx
+.. tabs::
+
+    .. group-tab:: Pure Python
+
+        .. literalinclude:: ../../examples/tutorial/external/keyword_args_call.pxd
+            :caption: keyword_args_call.pxd
+        .. literalinclude:: ../../examples/tutorial/external/keyword_args_call.py
+            :caption: keyword_args_call.py
```

Comment:
I don't think it's didactically helpful to use the module's own .pxd file for the declaration. At least not without an elaborate explanation of how it's different from other .pxd files and what it does to your module and its namespace.

You can just name it `strstr.pxd`. That's all it provides, and it (hopefully) suggests naming .pxd files after what they provide.

Reply:
done.